source/content/glossar/bibliothek/geocat-examples/distribution-download-url.rst

```diff
@@ -1,5 +1,5 @@
 .. code-block:: xml
-    :caption: Example of mapping to dcat:downloadURL for a Distribtution with protocol "WWW:DOWNLOAD:INTERLIS"
+    :caption: Example of mapping to ``dcat:downloadURL`` for a Distribution with protocol "WWW:DOWNLOAD:INTERLIS"
```

Comment:
just noted that in the example above you corrected the capitalization of "Distribution" and in this example you did not. Maybe it should be "distribution" here as well. Same holds true for the examples below.